### ISSUE TITLE
Learning rate for ResNet 110

### ISSUE THREAD

**Issue by jakesokol1** [akamaster/pytorch_resnet_cifar10#21]
I am trying to train resnet110. After the first epoch, the learning rate does not return to .1, and it stays at .01. I believe this is not intended.

**Comment by jakesokol1** [comment]
I believe this yields expected behavior (if it is actually desired that only the first epoch have learning rate of .01).


    for epoch in range(args.start_epoch, args.epochs):
        if epoch == 1:
            for param_group in optimizer.param_groups:
                param_group['lr'] *= 10
`
